### DISCUSSION TITLE
Intstall Koreader in Remarkable 1 with the latest version 3.9

### DISCUSSION THREAD

**Discussion by Muathics** [koreader/koreader#11382] [Q&A] [answered]
It seems that Toltec only support version 2.x and the manual installation only adapt for Remarkable 2. So how do I Intstall Koreader in Remarkable 1 with the latest version 3.9?

**Comment by pazos** [comment]
Manual installation is the same on both devices. RM2FB is mandatory for rm2 and not required for rm1.

Please do understand than tinkering with a device requires some knowledge and nobody here can assist you in case you screw something.

Also, AFAIK, nobody tested the program in fw 3.9, so there're some chances it won't work without changes

**Reply by Muathics** [reply, marked as answer]
Thanks for your reply. I misunderstand it. It works well for version 3.9.

**Reply by onqun** [reply]
I have rm2 would you explain how can i install ko readrr

**Reply by benoit-pierre** [reply]
Check the [wiki page](https://github.com/koreader/koreader/wiki/Installation-on-ReMarkable) for ReMarkable.

**Reply by Commodore64user** [reply]
> screw something.

screw something UP, there is a remarkable difference between the two. ;)
